code/game/objects/items/devices/flashlight.dm

```diff
@@ -340,7 +340,7 @@
 /obj/item/flashlight/flare/torch
 	name = "torch"
 	desc = "A torch fashioned from some leaves and a log."
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
they should be normal at least, you make them with an entire log of wood and they look pretty big in hand